people/zhiyuan-jia.md

```diff
@@ -0,0 +1,7 @@
+Zhiyuan Jia
+-----------
+
+![](photos/zhiyuan-jia.jpg)
+
+*[LinkedIn](https://www.linkedin.com/in/zhiyuan-jia-42a961148/?trk=uno-choose-ge-no-intent&dl=no)
```

Comment:
Whoops! Did you mean to add bullets or italics?

Reply:
I will fix that! thx